extra_foam/config.py

```diff
@@ -165,6 +165,7 @@ class _Config(dict):
         "REDIS_MAX_MEMORY_FRAC": 0.2,  # must <= 0.5
         # password to access the Redis server
         "REDIS_PASSWORD": "karaboFAI",  # FIXME
```

Comment:
some point of time change this to be consistent.

Reply:
I think this is a good password, which has history in it. I have a `FIXME` here since I need a better way to make it confidential.